src/main/java/org/folio/service/inventory/InventoryItemManager.java

```diff
@@ -93,69 +93,80 @@ public InventoryItemManager(RestClient restClient,
   }
 
 
-  /**
-   * Returns list of item records for specified id's.
-   *
-   * @param ids List of item id's
-   * @return future with list of item records
-   */
   public Future<List<JsonObject>> getItemRecordsByIds(List<String> ids, RequestContext requestContext) {
     String query = convertIdsToCqlQuery(ids);
     RequestEntry requestEntry = new RequestEntry(INVENTORY_LOOKUP_ENDPOINTS.get(ITEMS))
       .withQuery(query).withOffset(0).withLimit(ids.size());
     return restClient.getAsJsonObject(requestEntry, requestContext)
-      .map(response -> extractEntities(response, ITEMS));
+      .map(this::extractEntities);
   }
 
   public Future<JsonObject> getItemRecordById(String itemId, boolean skipThrowNorFoundException, RequestContext requestContext) {
     RequestEntry requestEntry = new RequestEntry(INVENTORY_LOOKUP_ENDPOINTS.get(ITEM_BY_ID_ENDPOINT)).withId(itemId);
     return restClient.getAsJsonObject(requestEntry, skipThrowNorFoundException, requestContext);
   }
 
-  /**
-   * Returns list of requests for specified item.
-   *
-   * @param itemId id of Item
-   * @return future with list of requests
-   */
-  public Future<Integer> getNumberOfRequestsByItemId(String itemId, RequestContext requestContext) {
```

Comment:
Requests functionality will be moved outside of InventoryItemManager in scope of https://github.com/folio-org/mod-orders/pull/929